### ISSUE TITLE
[Interop] getMembers returns empty list for Strings

### ISSUE THREAD

**Issue by fniephaus** [oracle/truffleruby#1817]
#### Repro session
```
$ polyglot --shell --jvm
GraalVM MultiLanguage Shell 19.3.0
Copyright (c) 2013-2019, Oracle and/or its affiliates
  Python version 3.7.4
  Ruby version 2.6.2
python> import polyglot
python> x = polyglot.eval(language='ruby', string='"Test"')
python> x
Test
python> dir(x)
[] # NON-EMPTY LIST EXPECTED
python> x.length()
4 # WORKS AS EXPECTED
```

**Comment by eregon** [comment]
Thank you for the report.
@pitr-ch Could you take a look at this?

**Comment by pitr-ch** [comment]
@fniephaus thanks for the report. This is a behaviour for strings we had from the beginning. I know about the problem, there is more inconsistencies like this in our Interop. I am going through them and it will be all fixed during the winter. Is it something which is causing problems for you at the moment?

**Comment by fniephaus** [comment]
@pitr-ch thanks for the update. Our students use GraalVM for various experiments and these inconsistencies make it hard to work with TruffleRuby sometimes. This is not necessarily urgent, but it would be great if these inconsistencies could be addressed soon.

**Comment by pitr-ch** [comment]
It turned out I was confusing this problem with something else that was hard to fix. I have a fix for this issue and it should be merged in few days.

**Comment by fniephaus** [comment]
Great, thanks!

**Comment by pitr-ch** [comment]
Fixed in 6c7d440d791597e4311a283c5f15b54446d41251
